kolibri/content/management/commands/importchannel.py

```diff
@@ -65,12 +65,13 @@ def _transfer(self, method, channel_id, path=None):
 
         with filetransfer:
 
-            with self.start_progress(total=filetransfer.total_size) as progress_update:
+            with self.start_progress(total=filetransfer.total_size*1.1) as progress_update:
 
                 for chunk in filetransfer:
                     progress_update(len(chunk), progress_extra_data)
 
-        annotation.update_channel_metadata_cache()
+                channel_import.import_channel_from_local_db(channel_id)
+                progress_update(filetransfer.total_size*0.1, progress_extra_data)
```

Comment:
So I know in previous versions it ran `importchannel` and `importcontent` as separate commands, and it did separate progress for each -- it ran the progress bar from 0 to 100 first for the channel import, then started over from 0 and went up again for the content import. How does this relate to that? Will it go up to 10% now, and then reset to 0 for the content import?

Ideally/eventually, we'd have subtask progress, and could assign these types of percentages either in a rational or arbitrary way, to each of the phases/tasks, so we'd have a nice continuous overall progress bar and potentially a subtask progress bar as well.